primitives/xcm-bridge-hub-router/src/lib.rs

```diff
@@ -29,3 +33,18 @@ impl XcmChannelStatusProvider for () {
 		false
 	}
 }
+
+/// Current status of the bridge.
+#[derive(Clone, Decode, Encode, Eq, PartialEq, TypeInfo, MaxEncodedLen, RuntimeDebug)]
+pub struct BridgeState {
```

Comment:
@svyatonik 
I think, we should rename this to `BridgeHubState` and use bridge-hub instead of bridge here,
it brings a little bit confusion what is "bridge" and "bridge hub" here,
because we have "xcm router to bridge hub",
and also I would rename in router:
```
pub type Bridge<T: Config<I>, I: 'static = ()> = StorageValue<_, BridgeState, ValueQuery>;
pub type BridgeHub<T: Config<I>, I: 'static = ()> = StorageValue<_, BridgeHubState, ValueQuery>;
```

Reply:
hmm, how do you change it later?
```
pub type BridgeHub<T: Config<I>, I: 'static = ()> = StorageMap<_, LaneId, BridgeHubState, ValueQuery>;
```

Reply:
maybe `BridgeHubLaneState`?

Reply:
Yes, it'll be the map in the future. So it isn't the bridge hub state - it is the state of single bridge that is served by single bridge hub. And at bridge hub it has a dedicated lane. And there'll be something like `type BridgeId = LaneId`. So sending chains (AH), where this router will be deployed, will only deal with "bridges" and "bridge hub". At bridge hub we'll deal with both bridges and their dedicated lanes.

Now we have the single lane, so I could rename it to `BridgeHubState` as you suggested. But in v2, it'll be the `BridgeState` anyway

Reply:
ok, no need to rename `BridgeState`,
"bridge" is too vague , I mean "bridge" word is everywhere here :D :D :D,
at least from the doc and terminology needs to be clear what is "bridge" vs "lane" in this context